Makefile

```
@@ -188,21 +188,44 @@ release: dist ## package and upload a release
 	twine upload dist/*
 
 dist: clean ## builds source and wheel package
+	$(MAKE) clean
 	pip install setuptools
-	python3 setup.py sdist
-	python3 setup.py bdist_wheel
+	python setup.py sdist
+	python setup.py bdist_wheel
 	ls -l dist
 
+
 ## install the package to the active Python's site-packages
 # Note that instead of --force-reinstall, we uninstall and then install, because
 # --force-reinstall also reinstalls all deps. And if we also used --no-deps, then the
 # deps wouldn't be installed the first time.
 install: dist
 	pip uninstall -y shiny
-	python3 -m pip install dist/shiny*.whl
+	python -m pip install dist/shiny*.whl
+ci-install-wheel: FORCE
+	$(MAKE) dist
```

I think this should work, but please test:

```suggestion
ci-install-wheel: FORCE dist
```